src/quoteservice/app/routes.php

```diff
@@ -26,12 +26,20 @@ function calculateQuote($jsonObject): float
             throw new \InvalidArgumentException('numberOfItems not provided');
         }
         $numberOfItems = intval($jsonObject['numberOfItems']);
-        $quote = round(8.90 * $numberOfItems, 2);
+        $costPerItem = rand(400, 1000)/10;
+        $quote = round($costPerItem * $numberOfItems, 2);
 
         $childSpan->setAttribute('app.quote.items.count', $numberOfItems);
         $childSpan->setAttribute('app.quote.cost.total', $quote);
 
         $childSpan->addEvent('Quote calculated, returning its value');
+
+        //manual metrics
+        static $counter;
+        $counter ??= Globals::meterProvider()
+            ->getMeter('quotes')
+            ->createCounter('quotes', 'quotes', 'number of quotes calculated');
+        $counter->add(1, ['number_of_items' => $numberOfItems, 'total_cost' => $quote]);
```

Comment:
The `total_cost` attribute here will create a cardinality explosion on the metric. One could also argue the `number_of_items` tag could do the same, but I feel like that number won't go too wild, and it makes sense to leave it so people can see how you add tags to metrics.

Reply:
removed total cost attribute